practice-two/src/index.html

```diff
@@ -75,7 +75,60 @@ <h3 class="banner-section__title">We’re Bring It To The New Level</h3>
   <!--END HEADER-->
   <main>
     <!-- START ADVANTAGES SECTION -->
-    <section></section>
+
+    <section class="advantages-section">
+      <div class="container advantages">
+        <hgroup class="advantages__header">
+          <h2 class="heading-section advantages__header-text">Stunning Results</h2>
+          <p class="text-normal advantages__header-text">Just look to the statistics we have generate the past year.</p>
+        </hgroup>
+        <div class="advantages__content">
+          <div class="advantages__items">
+            <div class="advantages__items-wrapper">
```

Comment:
No need use a div element to wrap here. You only need to use display flex and justify-content center for the advantages__items element